packages/gatsby-plugin-sitemap/src/internals.js

```diff
@@ -29,6 +29,10 @@ export const runQuery = (handler, query, excludes, pathPrefix) =>
       return page
     })
 
+    if (!r.data.site.siteMetadata.siteUrl) {
+      throw new Error(`SiteMetaData 'siteUrl' property is required`)
```

Comment:
What do you think about linking to https://www.gatsbyjs.org/packages/gatsby-plugin-sitemap/#how-to-use to see example configuration (to provide more context to error)

Reply:
Looked around for errors that reference links to docs, but didn't see any (swear have before though)?  If there is a standard format for this please let me know and I'll update it.